libr/debug/debug.c

```diff
@@ -293,6 +296,7 @@ R_API RDebug *r_debug_new(int hard) {
 	dbg->num = r_num_new (r_debug_num_callback, dbg);
 	dbg->h = NULL;
 	dbg->threads = NULL;
+	dbg->hitinfo = 1;
```

Comment:
Ok

> On 01 Sep 2016, at 16:55, Tiago Gasiba <notifications@github.com> wrote:
> 
> In libr/debug/debug.c:
> 
> > @@ -293,6 +296,7 @@ R_API RDebug *r_debug_new(int hard) {
> >  	dbg->num = r_num_new (r_debug_num_callback, dbg);
> >  	dbg->h = NULL;
> >  	dbg->threads = NULL;
> > +	dbg->hitinfo = 1;
> could be bool, could be integer if in the future several "info levels" are defined. this is why i kept it at int; if not ok, if you think otherwise, i can change it to bool
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>